install_requirements.py

```diff
@@ -42,6 +42,10 @@
 subprocess.check_call(pip_call + ["uninstall", "depthai", "--yes"])
 subprocess.check_call(pip_package_install + ["-r", "requirements.txt"], cwd=scriptDirectory)
 
+# arm version is installed by homebrew in installation script
+if not is_pi:
+    subprocess.check_call(pip_call + ["install", "pyqt5"])
```

Comment:
Needed? pyqt5 is already specified in requirements.txt

Reply:
Needed, I deleted PyQt5 from requirements. It causes problem for m1 chips because PyQt5 for mac silicon is not available in pip

Reply:
@PetrNovota 
You can out it based on arch, but that works as well. 
Also, does is_pi then does the right thing for all the others?
Shouldn't it be `if not is_m1`?

Reply:
I'd rename `is_pi` to `is_arm`

Reply:
I agree that is_py == True when arch is arm*, that should hold true for mac silicon as well as for other arm platforms

Reply:
ill rename it to is_arm